### PR TITLE
M3-5732: Improve styling consistency for backup auto enrollment

### PR DESCRIPTION
**Pull request by jdamore-linode**
## Description

**What does this PR do?**
Improves the consistency of the styling for the "enable now" link that is found in the "Backup Auto Enrollment" section of the Account Settings page.

## How to test

**What are the steps to reproduce the issue or verify the changes?**
Navigate to `/account/settings` and confirm that the "enable now" link style is consistent with other links throughout Cloud Manager.

### CODE REVIEW

File: packages/manager/src/features/Account/AutoBackups.tsx [hunk 1]
```diff
@@ -19,6 +18,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontSize: '0.8em',
     marginLeft: theme.spacing(1) / 3,
   },
+  enableBackupsButton: {
+    ...theme.applyLinkStyles,
+    fontSize: 14,
```
Comment (DevDW) [inline packages/manager/src/features/Account/AutoBackups.tsx:23]: let's go with `0.875rem` here -- equivalent sizes but our conventional best practice is to use `rem` over `px`and `em`

File: packages/manager/src/features/Account/AutoBackups.tsx [hunk 2]
```diff
@@ -91,15 +94,14 @@ const AutoBackups: React.FC<Props> = (props) => {
           <Grid item>
             <Typography variant="body1" className={classes.footnote}>
               For existing Linodes without backups,&nbsp;
-              <Button
-                compact
-                buttonType="secondary"
+              <button
                 data-qa-backup-existing
                 onClick={openBackupsDrawer}
                 title="enable now"
```
Comment (DevDW) [inline packages/manager/src/features/Account/AutoBackups.tsx:100]: thoughts on removing the `title`? Right now if you hover over the button, you'll see that little pop-up that says "enable now". If we remove the `title`, that will go away.

**Review by DevDW — APPROVED** [review]
LGTM, left a couple of small comments/tweaks 👍

File: packages/manager/src/features/Account/AutoBackups.tsx (hunk 2 above)
Comment (jdamore-linode) [reply packages/manager/src/features/Account/AutoBackups.tsx:100]: Agreed, I think it's pretty redundant as-is. For screen reader users, do you think it'd be preferable to remove the title, or to change the title to something more useful or complete, like "Enable Backup Auto Enrollment"?

Comment (DevDW) [reply packages/manager/src/features/Account/AutoBackups.tsx:100]: Good question -- I'm leaning towards the latter, but we are inconsistent in the codebase about having the `title` attribute. @tiffwong what are your thoughts on this?

Comment (tiffwong) [reply packages/manager/src/features/Account/AutoBackups.tsx:100]: I did some research and the `title` attribute is not helpful and gets missed a lot on screen readers.

**Review by tiffwong — APPROVED** [review]
Great job on this!

**Pull request merged**